core/blockchain.go

```diff
@@ -2294,6 +2294,7 @@ func (bc *BlockChain) indexBlocks(tail *uint64, head uint64, done chan struct{})
 			from = head - bc.txLookupLimit + 1
 		}
 		rawdb.IndexTransactions(bc.db, from, head+1, bc.quit)
+		rawdb.WriteTxIndexTail(bc.db, 0)
```

Comment:
The tail flag will be set in `rawdb.IndexTransactions`. And also the tail will not be 0 in case from is not 0

Reply:
> The tail flag will be set in `rawdb.IndexTransactions`. And also the tail will not be 0 in case from is not 0

It should be set as ```from```